spec/models/family_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Family do
+  describe "validations" do
+    subject { Family.new(last_name: nil) }
+
+    context "when creating a new family with no last_name is not valid" do
+      it { should_not allow_value(nil).for(:last_name)}
```

Comment:
We prefer to use the new `is_expected.to` syntax over the old `should`

Reply:
Since you have a describe block with the description of "validations", I think the nested context blocks add more noise then being useful.

Also, a suggestion to use the following format:
```
it { is_expected.to validate_presence_of(:last_name) }
```

Reply:
Thank Elle, I will remove the subject and modify the test structure.